WordPress/src/main/java/org/wordpress/android/push/NotificationsProcessingService.java

```diff
@@ -554,6 +563,10 @@ private void approveComment() {
                 return;
             }
 
+            // keep the local CommentId, we'll use it later to know whether to trigger the end processing notification
+            // or not
+            keepRemoteCommentIdForPostProcessing(comment.getRemoteCommentId());
```

Comment:
Should we move this call in `performRequestedAction` to be sure to catch comment like and other quick actions that may generate notifications?

Reply:
I was using it like that (added the `keepRemoteCommentIdForPostProcessing()` call in all 3 possible actions in the Service) but then decided to leave it only for the FluxC `PUSH` action as it was where the reported case was happening, but I think you're right, we should move it back and make it work for all cases. Will do and comment back here 👍